### ISSUE TITLE
clearNotifyBlock should not `setNotify: NO` if disconnected

### ISSUE THREAD

**Issue by matsoft90** [Raizlabs/RZBluetooth#74]
@cpatterson-lilly @KingOfBrian I am having a small problem... I have a peripheral that always automatically reconnects if I manually call `cancelConnectionWithCompletion:`. I have `maintainConnection` false for this peripheral right now. Since `maintainConnection` is false the `connectWithCompletion:` method is not called from within `connectionEvent:error:`

I see the following in the logs

```
RZBLogLevelCommand - cancelPeripheralConnection: <PERIPHERAL UUID>
RZBLogLevelDelegate - centralManager:didDisconnectPeripheral:error: - <CBCentralManager: 0x17007dd80> <PERIPHERAL UUID> (null)
RZBLogLevelCommand - connectPeripheral:<PERIPHERAL UUID> options:(null)
RZBLogLevelDelegate - centralManager:didConnectPeripheral: - <CBCentralManager: 0x17007dd80> <PERIPHERAL UUID>
```

I must mention however that I have another peripheral that I do not see the same behavior on. The same peripheral behaves normally using CoreBluetooth directly.

It appears that after calling disconnect `executeCommandWithContext:error:` is still called on an `RZBConnectCommand` instance

**Comment by matsoft90** [comment]
It appears to be happening inside `connectedPeripheralForUUID:triggeredByCommand` because of a `RZBNotifyCharacteristicCommand` I do have notify enabled for a couple characteristics. But even then a disconnection should cancel that as well.

**Comment by matsoft90** [comment]
I was calling `clearNotifyBlock` on my disconnected peripheral. I'm sorry to bother you both.

**Comment by KingOfBrian** [comment]
No problem @matsoft90. I'm still interested in the usability here and this still feels a bit off. You called `clearNotifyBlock` on a disconnected peripheral, and it caused a reconnection? If so, that's not entirely intuitive, and a valid bug in my opinion!

**Comment by matsoft90** [comment]
Yes @KingOfBrian that's exactly what I did. And you're right. It isn't very intuitive. A solution could be to not call the reconnect method if the command is of type `RZBNotifyCharacteristicCommand`. I'll see if I can make this quick change and create a PR.

**Comment by matsoft90** [comment]
@KingOfBrian I created a PR though I'm not sure if this is how you wanted it implemented.

**Comment by KingOfBrian** [comment]
This should be addressed in #95 !
